CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ if(MSVC)
   if(USE_F16C)
     message("F16C instruction set is not yet supported for MSVC")
   endif()
-  set(CMAKE_C_FLAGS "/MP")
+  # set(CMAKE_C_FLAGS "/MP")
```

Comment:
Why this change?

Reply:
/MP caused some errors earlier, I will take a look later and add this back if it works properly.

Reply:
/MP causes error if the compiler is Clang-cl. I will enable the option if the compiler is MSVC.